### DISCUSSION TITLE
Is there a way to group collections in cp menu?

### DISCUSSION THREAD

**Discussion by steffenschmidt** [statamic/cms#5299] [Q&A] [answered]
Hi there,

is there a way to group the collections in the control panel menu? I have a lot of collections which could be grouped for better view:

Example for status quo:
```
Collections
- Clients
- Coupons
- Dates
- Locations
- Orders
- Pages
- Products
- Testimonials
```

Example I would like to have:
```
Collections
- Pages
- Testimonials
- Events
  - Dates
  - Locations
- Shop
  - Clients
  - Coupons
  - Orders
  - Products
```

Currently I solved this by naming the collections with a prefix (e.g. `Shop Products`) but it would be nice to have a structured view.

Would that be a feature request or is there anything misconfigured from my side?

**Comment by ryanmitchell** [comment, marked as answer]
Yep you can do that - I do it with Simple Commere when we use it - see here:
https://github.com/thoughtco/simple-commerce-nav/blob/main/src/ServiceProvider.php
